javaslang-benchmark/src/test/java/javaslang/BenchmarkPerformanceReporter.java

```diff
@@ -162,15 +171,15 @@ public void print() {
         private void printHeader() {
             final String alternativeImplHeader = alternativeImplementations.map(altImpl -> padRight(altImpl, altImplColSize(altImpl))).mkString("  ");
             final String header = String.format("%s  %s  %s  %s  %s  %s  ±%s %s  %s",
-                    padLeft("Target", groupSize),
-                    padLeft("Operation", nameSize),
-                    padLeft("Impl", implSize),
-                    padRight("Params", paramKeySize),
-                    padRight("Count", countSize),
-                    padRight("Score", scoreSize),
-                    padRight("Error", errorSize),
-                    padRight("Unit", unitSize),
-                    alternativeImplHeader
+                                                padLeft("Target", groupSize),
+                                                padLeft("Operation", nameSize),
+                                                padLeft("Impl", implSize),
+                                                padRight("Params", paramKeySize),
+                                                padRight("Count", countSize),
+                                                padRight("Score", scoreSize),
+                                                padRight("Error", errorSize),
+                                                padRight("Unit", unitSize),
+                                                alternativeImplHeader
```

Comment:
Not sure what you mean, please check the separated PR if you think the formatting is ok there: https://github.com/javaslang/javaslang/pull/1558/files